spec/factories.rb

```diff
@@ -0,0 +1,14 @@
+include ActionDispatch::TestProcess
+FactoryGirl.define do
+  factory :post do
+    image { fixture_file_upload("#{Rails.root}/spec/support/fixtures/giraffe-dog.jpg", 'image/jpeg')  }
```

Comment:
Metrics/LineLength: Line is too long. [103/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)